package.json

```diff
@@ -817,7 +818,7 @@
         },
         {
           "command": "microsoft.powerplatform.pages.preview-site",
-          "when": "microsoft.powerplatform.pages.siteRuntimePreviewEnabled"
+          "when": "microsoft.powerplatform.pages.siteRuntimePreviewEnabled && workspaceFolderCount == 1 && !isWeb"
```

Comment:
`workspaceFolderCount == 1` is this some OOB config?

Reply:
Yes, https://code.visualstudio.com/api/references/when-clause-contexts